Cocoapods Example/PortalSwift/ViewController.swift

```diff
@@ -570,7 +570,7 @@ class ViewController: UIViewController, UITextFieldDelegate {
   func sendTransaction(ethEstimate: String) {
     let payload = ETHTransactionPayload(
       method: ETHRequestMethods.SendTransaction.rawValue,
-      params: [ETHTransactionParam(from: self.portal?.address ?? "", to: self.sendAddress?.text ?? "", gas: ethEstimate, gasPrice: ethEstimate, value: "0x10", data: "")]
```

Comment:
This is getting deleted because we were setting the `gas` field to `gasPrice` which is wrong and is now being caught because we are parsing the `gas` field